requirements.txt

```diff
@@ -6,3 +6,12 @@ psycopg2==2.8.3
 nose==1.3.7
 requests==2.22.0
 coverage==4.5.4
+pillow==6.1.0
+boto3==1.9.227
+botocore==1.12.227
+docutils==0.15.2
+jmespath==0.9.4
+python-dateutil==2.8.0
+s3transfer==0.2.1
+wtforms==2.2.1
+python_slugify==3.0.4
```

Comment:
Are we using all of these new libraries now?  I'd prefer keeping them to a minimum, especially `wtforms`.

Please comment here confirming what libraries are needed from this list and prune the ones that aren't definitely needed.

Reply:
I'm cool with wtforms as I see you're using this library extensively now.